lib/annotate_rb/parser.rb

```diff
@@ -35,11 +35,11 @@ def self.parse(args, existing_options = {})
     }.freeze
 
     def initialize(args, existing_options)
-      @args = args
+      @args = args.clone
```

Comment:
Before this change, `@args` was a reference to `ARGV`. This resulted in `parse_command` and `parser.parse!` to modify `ARGV`.